common/network-common/src/main/java/org/apache/spark/network/TransportContext.java

```diff
@@ -122,16 +122,14 @@ public TransportContext(
     this.closeIdleConnections = closeIdleConnections;
     this.isClientOnly = isClientOnly;
 
-    synchronized(TransportContext.class) {
-      if (chunkFetchWorkers == null &&
-          conf.getModuleName() != null &&
-          conf.getModuleName().equalsIgnoreCase("shuffle") &&
-          !isClientOnly) {
-        chunkFetchWorkers = NettyUtils.createEventLoop(
-            IOMode.valueOf(conf.ioMode()),
-            conf.chunkFetchHandlerThreads(),
-            "shuffle-chunk-fetch-handler");
-      }
+    if (chunkFetchWorkers == null &&
```

Comment:
No need to check for null; it always is here

Reply:
yes i think this was more of a defensive check when static was around will remove thanks